test/FastTests/Sparrow/Encoder3GramTests.cs

```diff
@@ -33,6 +33,14 @@ private struct StringKeys : IReadOnlySpanEnumerator, ISpanEnumerator
 
             public int Length => _values.Length;
 
+            public bool IsNull(int i)
+            {
+                if (i < 0 || i >= Length)
+                    throw new ArgumentOutOfRangeException();
```

Comment:
Do we need it, the `_values` already does this, no?

Reply:
Not really